app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -439,10 +448,12 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
         <WarningMessage
           additionalDescription={''}
           description={
-            'Providing liquidity is risky and could result in near total loss. It is important to withdraw liquidity before the event occurs and to be aware the market could move abruptly at any time.'
+            currentToken.exists
+              ? `An identical market with ${currentToken.token} as a currency exists already. Please use a different currency or  provide liquidity for the`
+              : 'Providing liquidity is risky and could result in near total loss. It is important to withdraw liquidity before the event occurs and to be aware the market could move abruptly at any time.'
           }
-          href={DOCUMENT_FAQ}
-          hyperlinkDescription={'More Info'}
+          href={currentToken.exists ? `${realitioBaseUrl}/app/#!/question/${loadedQuestionId}` : DOCUMENT_FAQ}
```

Comment:
This links to the reality.eth question, but we'd want to link to the omen market instead.